src/main/java/com/convallyria/taleofkingdoms/client/gui/entity/citybuilder/CityBuilderTierOneGui.java

```diff
@@ -0,0 +1,209 @@
+                    TaleOfKingdoms.getAPI().getClientPacketHandler(Packets.CITYBUILDER_ACTION)
+                            .handleOutgoingPacket(player, entity.getId(), CityBuilderAction.GIVE_64_STONE);
+                    return;
+                }
+
+                entity.give64stone(player);
+            }).positioning(Positioning.relative(80, 30)).sizing(Sizing.fixed(100), Sizing.fixed(20))
+        );
+
+        inner.child(
+            Components.label(Text.translatable("menu.taleofkingdoms.citybuilder.wood_amount"))
+                    .positioning(Positioning.relative(76, 40))
+        );
+
+        final float oakWoodPercent = oakWoodCount.get() * (100f / 320f);
+        inner.child(this.woodBar = (BarWidget) new BarWidget(0, 0, 100, 12, oakWoodPercent / 100)
+                .positioning(Positioning.relative(80, 45))
+                .tooltip(Text.literal(oakWoodCount.get() + " / 320")));
+
+        inner.child(
+                Components.label(Text.translatable("menu.taleofkingdoms.citybuilder.stone_amount"))
+                        .positioning(Positioning.relative(76, 53))
+        );
+
+        final float cobblestonePercent = cobblestoneCount.get() * (100f / 320f);
+        inner.child(this.stoneBar = (BarWidget) new BarWidget(0, 0, 100, 12, cobblestonePercent / 100)
+                .positioning(Positioning.relative(80, 57))
+                .tooltip(Text.literal(cobblestoneCount.get() + " / 320")));
+
+        inner.child(
+            this.fixWholeKingdomButton = (ButtonComponent) Components.button(Text.translatable("menu.taleofkingdoms.citybuilder.fix_kingdom"), c -> {
+                if (MinecraftClient.getInstance().getServer() == null) {
+                    TaleOfKingdoms.getAPI().getClientPacketHandler(Packets.CITYBUILDER_ACTION)
+                            .handleOutgoingPacket(player, entity.getId(), CityBuilderAction.FIX_KINGDOM);
+                    return;
+                }
+
+                entity.fixKingdom(player, kingdom);
+            }).tooltip(List.of(
+                  Text.literal("Repairing the kingdom costs:"),
+                  Text.literal(" - 320 oak wood"),
+                  Text.literal(" - 320 cobblestone")
+            )).positioning(Positioning.relative(80, 70)).sizing(Sizing.fixed(100), Sizing.fixed(20))
+        );
+
+        // Price list
+        inner.child(
+            Components.button(Text.translatable("menu.taleofkingdoms.citybuilder.price_list"), c -> {
+                MinecraftClient.getInstance().setScreen(new CityBuilderPriceListGui(player, entity, instance));
+            }).positioning(Positioning.relative(80, 80)).sizing(Sizing.fixed(100), Sizing.fixed(20))
+        );
+
+        // Actually starts at 70, first has addition of +20
+        int currentY = 15;
+        int currentX = 10;
+        final int maxPerRow = 7;
+        int currentRow = 0;
+        for (BuildCosts build : BuildCosts.values()) {
```

Comment:
What happens if the player tries to build something that's for tier 2+? need to disable invalid tier builds